src/test/ui/consts/miri_unleashed/enum_discriminants.rs

```diff
@@ -1,6 +1,9 @@
-// compile-flags: -Zunleash-the-miri-inside-of-you
+// compile-flags: -Zunleash-the-miri-inside-of-you -Awarnings
```

Comment:
I would prefer not to do this so that we can see changes in diffs and whatnot.

Reply:
The "const check ignored" annotations were only added last week since they are now mandatory in all UI tests. They are unrelated to the actual purpose of the test, they just add noise (to both the test itself and PRs that work on const qualification). Ignoring warnings was explicitly mentioned as an opt-out path for tests like this.